bears/hypertext/HTMLLintBear.py

```diff
@@ -27,10 +27,13 @@ class HTMLLintBear:
     CAN_DETECT = {'Syntax', 'Formatting'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file,
+    def create_arguments(filename, file, config_file, use_spaces: bool,
                          htmllint_ignore: typed_list(str)=()):
         """
         :param htmllint_ignore: List of checkers to ignore.
+        :param use_spaces: True if spaces are to be used instead of tabs.
         """
         ignore = ','.join(part.strip() for part in htmllint_ignore)
+        if not use_spaces:
+            ignore += ',tabs'
```

Comment:
what if `htmllint_ignore` is empty? then your string has a leading comma:
```
>>> ignore
',tabs'
```
I think cleanest is to embed this into the `join`, maybe like this:
```python
additional_ignore = []
if not use_spaces:
    additional_ignore.append('tabs')
ignore = ','.join(part.strip() for part in htmllint_ignore + additional_ignore)
```

Reply:
I will make the necessary changes :+1: